documentation/endpoints/posts.md

```diff
@@ -18,6 +18,8 @@ POST /api/v2/posts
     The reason why the user participated.
   - **caption**: (string).
     Corresponding caption for the post.
+  - **status**: (string).
```

Comment:
Did you have to do anything to make sure this got stored?

Reply:
we actually already had that functionality! Just wasn't in the documentation so added it :)

Reply:
sweet!